microsoft-edge/devtools-guide-chromium/whats-new/2023/08/devtools-116.md

```diff
@@ -0,0 +1,115 @@
+<!--
+* [Dec. 2023 blog post about Focus Mode]() - todo: link -->
+
+
+<!-- ====================================================================== -->
+## React Developer Tools extension retains state of tools in Focus Mode
+
+<!-- Subtitle: The React Developer Tools extension retains the state of tools while in Focus Mode. -->
+<!-- todo: write actual subtitle -->
+
+Prior to Microsoft Edge 116, if you were in Focus Mode while using the React Developer Tools extension, the state of a tool in DevTools wasn't maintained when you switched to another tool and then switched back.  This issue has been resolved.
+
+See also:
+* [React Developer Tools](https://microsoftedge.microsoft.com/addons/detail/react-developer-tools/gpphkfbcpidddadnkolkpfckpihlkkil) - extension at the Edge Add-ons store
+* [What's New entries about Focus Mode](https://learn.microsoft.com/search/?scope=Microsoft%20Edge&terms=focus%20mode)
+<!--
+* [Dec. 2023 blog post about Focus Mode]() - todo: link -->
+
+
+<!-- ====================================================================== -->
+## Containment Diff view in Memory tool when multiple heap snapshots
+
+<!-- Subtitle: __ -->
+<!-- todo: need Subtitle sentence -->
+
+When there's more than one heap snapshot, the "Containment Diff" view will be available in the View dropdown:
+
+![The Containment Diff view in the Memory tool](./devtools-116-images/new-view-memory-panel.gif)
+
+The Containment Diff view shows only the new objects, deleted objects, or objects in the retaining chain that will eventually lead to either a new or deleted node.  The "." symbol means that it's a node in the retaining chain.  Expanding the view displays the node that's new or deleted.
+
+See also:
+* [Record heap snapshots using the Memory tool](../../../memory-problems/heap-snapshots.md)
+
+
+<!-- ====================================================================== -->
+## Added an icon in the Recorder tool's tab
+
+<!-- Subtitle: __ -->
+<!-- todo: need Subtitle sentence -->
+
+<!-- todo: can't just mention update of Recorder tool, since the Recorder tool has never been mentioned or introduced as new in What's New.  Need What's New entry "Microsoft Edge v123 added the Recorder tool from the Chromium engine" -->
+
+Before Microsoft Edge 116, there was no icon in the **Recorder** tool's tab on the Activity bar:
+
+![Recorder panel with no icon](./devtools-116-images/recorder-before.png)
+
+Microsoft Edge 116 adds an icon in the **Recorder** tool's tab on the Activity bar:
+
+![Recorder panel with new icon](./devtools-116-images/recorder-after.png)
+
+See also:
+* [Record, replay, and measure user flows (Recorder tool)](https://developer.chrome.com/docs/devtools/recorder)
+* [Recorder features reference](https://developer.chrome.com/docs/devtools/recorder/reference)
+* [Customize the Recorder with extensions](https://developer.chrome.com/docs/devtools/recorder/extensions)
+<!-- todo: link to local articles -->
```

Comment:
create local articles and then link to them.  until then, link to Chrome docs here?
@captainbrosset @leahmsft

Reply:
do in What's New 120 PR